build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.0'
+        classpath 'com.android.tools.build:gradle:3.1.0-alpha05'
```

Comment:
Would prefer not to use alpha build tools here.

Reply:
Ok, changed